project/Deps.scala

```diff
@@ -8,14 +8,14 @@ object Deps {
 
   // process lifecycle
```

Comment:
Should we update `addSbtPlugin("com.twitter" %% "scrooge-sbt-plugin" % "18.4.0")` in `plugin.sbt` as well?

Reply:
good catch